### DISCUSSION TITLE
Muse Sounds documentation

### DISCUSSION THREAD

**Discussion by ariutti** [musescore/MuseScore#15940] [Q&A]
Hello everyone, 
I'll get straight to the point: what are MuseScore's virtual instruments and how do they work?

What data do they receive when playing the score (probably MIDI but not only, as @Tantacrul  points out in [this](https://www.youtube.com/watch?v=Qct6LKbneKQ&feature=youtu.be) on video).

As far as I can see they are a bit like the "`.sfz`" format (have extension `.sts` and `.spx`).

Do they use samples from other available free sample library (like [VCSL](https://vis.versilstudios.com/vcsl.html), [VSCO-2-CE](https://vis.versilstudios.com/vsco-community.html) or [SSO](https://github.com/peastman/sso) or they use specially recorded sounds?

Is there, or will be available, any documentation on the format they use, or on the `MuseSampler` software component (which, I believe, is the sampler that can interpret them)?

I was blown away by the way they sound! Even if they still need some fine-tuning, they are something stunning. 
I am super curious to know more about how they work and I was not able to find details inside the [handbook](https://musescore.org/en/handbook/4/soundfonts).

**Comment by YuenSzeHong** [comment]
If possible, I would like to be able to map MuseSounds instructions to VST3 instructions so VST3 orchestral libs can work with MuseScore better
